modules/tinymce/src/plugins/table/main/ts/ui/UiUtils.ts

```diff
@@ -17,10 +17,16 @@ interface Item {
 const onSetupToggle = (editor: Editor, formatName: string, formatValue: string) => {
   return (api: Toolbar.ToolbarMenuButtonInstanceApi) => {
     const boundCallback = Singleton.unbindable();
+    const isNone = Strings.isEmpty(formatValue);
 
     const init = () => {
-      api.setActive(editor.formatter.match(formatName, { value: formatValue }));
-      const binding = editor.formatter.formatChanged(formatName, api.setActive);
+      // If value is empty (A None-entry in the list), check if the format is not set at all. Otherwise, check if the format is set to the correct value.
+      const setActive = (matched: boolean) =>
+        api.setActive(isNone ? !matched : matched);
+
+      setActive(editor.formatter.match(formatName, { value: formatValue }, undefined, isNone));
+      // formatChanged currently incorrectly highlights items. TODO TINY-7713
```

Comment:
```suggestion
      // TODO: TINY-7713: formatChanged doesn't currently handle formats with dynamic values so this will currently cause all items to show as active
```
TODO should always be at the start, not at the end.

Also, I wonder if we should just have this commented out for now? As what's better not updating or showing all items as active?

Reply:
I commented it out. It's not likely to occur to begin with, so "Not updating" is probably better than "Updating incorrectly", as you said.